### DISCUSSION TITLE
How to use template in the templates folder?

### DISCUSSION THREAD

**Discussion by chipit24** [remix-run/remix#3825] [Q&A] [answered]
I noticed there's a templates folder: https://github.com/remix-run/remix/tree/main/templates.

I tried to run `npx create-remix --template remix-run/cloudflare-workers` but got the following error:

```
🚨 The template could not be verified. Please double check that the template is a valid GitHub repository and try again.
```

I was expecting this, since [the docs](https://remix.run/docs/en/v1/pages/stacks#--template) mention I need to reference either the name of a stack in the remix-run GH org or a GH username/repo combo; no mention of referencing a specific folder in a GH repo.

Can these templates be used with the `create-remix` command, or are they only for reference? 

____

**Update:** Ah, `npx create-remix --template cloudflare-workers` worked, I should have tried that 🤦‍♂️ I guess my question now is, how does this work? Since it doesn't seem to be documented.

**Comment by machour** [comment, marked as answer]
The templates in `main/templates` are the non-stack templates (just the basic)

You'll get prompted to pick one if you chose `just the basics`, and it's all [documented](https://remix.run/docs/en/v1/other-api/dev#remix-create---template) in the remix-create documentation page
